builtin/analyzer.go

```diff
@@ -40,6 +40,9 @@ func (ba Analyzer) Analyze(env core.Env, form core.Any) (core.Expr, error) {
 	case Symbol:
 		return ResolveExpr{Symbol: f}, nil
 
+	case core.Vector:
+		return VectorExpr{Vector: f}, nil
```

Comment:
Don't you need to initialise the analyser field here ?🤔

Reply:
Ah yes, thanks for catching that.  I've fixed it and added regression tests.